SpacetimeDB.ClientSDK.csproj

```diff
@@ -5,11 +5,11 @@
     <LangVersion>9</LangVersion>
     <ImplicitUsings>disable</ImplicitUsings>
     <Nullable>enable</Nullable>
-    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <PackageId>SpacetimeDB.ClientSDK</PackageId>
     <Title>SpacetimeDB SDK</Title>
     <Authors>jdetter</Authors>
     <Company>Clockwork Labs</Company>
+    <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
```

Comment:
Maybe better to use `PackageLicenseExpression` so that it can nicely show up on NuGet pages?

Reply:
it sounds like there isn't an expression we could use.. and also it's currently intentional that we publish without a license, so I'm removing these

Reply:
Hm LICENSE.txt looks like a standard `Apache-2.0` SPDX?

Reply:
apparently we currently intentionally publish with no license

Reply:
(I assume since I've reverted these changes then `LICENSE.txt` won't be bundled in?)